mono/utils/mono-compiler.h

```diff
@@ -5,6 +5,15 @@
 #ifndef __UTILS_MONO_COMPILER_H__
 #define __UTILS_MONO_COMPILER_H__
 
+// These macros are duplicated in mono-compiler.h, mono-publib.h, glib.h for maximum reach and consistency.
```

Comment:
Why the duplication?

Reply:
The comment indicates but maybe inadequately— the headers are not always included and adding or reordering includes is unwarranted risk/change.